packages/template-retail-react-app/app/components/list-menu/index.jsx

```diff
@@ -65,7 +65,6 @@ const ListMenuTrigger = ({item, name, isOpen, onOpen, onClose, hasItems}) => {
                 onMouseOver={onOpen}
                 {...baseStyle.listMenuTriggerLink}
                 {...(hasItems ? {name: name + ' __'} : {name: name})}
-                {...(!hasItems ? baseStyle.listMenuTriggerLinkWithIcon : {})}
```

Comment:
Are we using this style anywhere else? If not, we should also remove it in the theme file

Reply:
It's not being used anywhere else, I will remove it!